prow/prowjobs/kubeflow/pipelines/kubeflow-pipelines-presubmits.yaml

```diff
@@ -190,6 +190,16 @@ presubmits:
       - image: python:3.7
         command:
         - ./v2/test/presubmit.sh
+  
+  - name: kubeflow-pipelines-manifests
+    cluster: build-kubeflow
+    decorate: true
+    run_if_changed: "^manifests/kustomize/.*$"
+    spec:
+      containers:
+      - image: gcr.io/google.com/cloudsdktool/cloud-sdk:latest
+        command:
+        - ./manifests/kustomize/hack/presubmit.sh
```

Comment:
I don't see the `presubmit.sh` file yet: https://github.com/kubeflow/pipelines/tree/master/manifests/kustomize/hack, are we going to add this file first?

Reply:
Ah I see the prerequisite PR: https://github.com/kubeflow/pipelines/pull/5390/files

Reply:
Hi @zijianjoy, I was being a little lazy, there are no PRs other than the current one that is changing kustomize manifests. So I think it's fine to add the config change first, as long as the manifest PR looks good to you.